src/jaxsim/api/kin_dyn_parameters.py

```diff
@@ -98,9 +98,7 @@ def build(model_description: ModelDescription) -> KynDynParameters:
         ]
 
         # Create a vectorized object of link parameters.
-        link_parameters = jax.tree_util.tree_map(
-            lambda *l: jnp.stack(l), *link_parameters_list
-        )
+        link_parameters = jax.tree.map(lambda *l: jnp.stack(l), *link_parameters_list)
```

Comment:
Can you please check what is the first JAX release that supports this? I suspect we need to bump the version we currently support.